tests/integration/targets/vmware_vm_storage_policy/tasks/main.yml

```diff
@@ -0,0 +1,115 @@
+# Test code for the vmware_tag CRUD Operations.
+# Copyright: (c) 2020, Dustin Scott <sdustin@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- when: vcsim is not defined
```

Comment:
You can also drop the conditional block. The CI, and `ansible-test` won't run vcsim tests by default anyway.

Reply:
Fixed in https://github.com/ansible-collections/vmware/pull/237/commits/1afea12cf2dec628d4f143528bbf1b455384be84